spec/models/entities/opportunity_spec.rb

```diff
@@ -30,7 +30,7 @@
 require File.expand_path(File.dirname(__FILE__) + '/../../spec_helper')
 
 describe Opportunity do
-  before { login }
+  let(:current_user) { FactoryGirl.create(:user) }
```

Comment:
Its weird that these tests want a user
```
  it "should be possible to create opportunity with the same name" do
    FactoryGirl.create(:opportunity, name: "Hello", user: current_user)
    expect { FactoryGirl.create(:opportunity, name: "Hello", user: current_user) }.to_not raise_error
  end
```

We should shift that into a context to explain it ("with a logged in user"); put it in the test case name or drop the user requirements